documentation/modules/managing/proc-cluster-recovery-volume.adoc

```diff
@@ -0,0 +1,218 @@
+// Module included in the following assembly:
+//
+// assembly-cluster-recovery-volume.adoc
+
+[id="cluster-recovery-volume_{context}"]
+= Recovering a cluster from persistent volumes
+
+This procedure describes how to recover a deleted cluster from persistent volumes (PVs).
+
+The procedure assumes that the Topic Operator is deployed.
+In situations such as these, the Topic Operator identifies that topics exist in Kafka, but the `KafkaTopic` resources do not exist.
+The `KafkaTopic` resources must be recovered before a cluster is started so that the topics are not deleted by the Topic Operator.
+
+If the Topic Operator is not deployed, you only need to recover the `PersistentVolumeClaim` (PVC) resources.
+
+.Before you begin
+
+In this procedure, it is essential that PVs are mounted into the correct PVC to avoid data corruption.
+A `volumeName` is specified for the PVC and this must match the name of the PV.
+
+For more information, see:
+
+* xref:ref-persistent-storage-{context}#pvc-naming[Persistent Volume Claim naming]
+* xref:ref-jbod-storage-{context}#jbod-pvc[JBOD and Persistent Volume Claims]
+
+NOTE: The procedure does not include recovery of `KafkaUser` resources, which must be recreated manually.
+
+.Procedure
+
+. Check information on the PVs in the cluster:
++
+[source,shell,subs="+quotes,attributes"]
+----
+kubectl get pv
+----
++
+Information is presented for PVs with data.
++
+Example output showing columns important to this procedure:
++
+[source,shell,subs="+quotes,attributes"]
+----
+NAME                                         RECLAIM POLICY
+pvc-5e9c5c7f-3317-11ea-a650-06e1eadd9a4c ... Retain
+pvc-5e9cc72d-3317-11ea-97b0-0aef8816c7ea ... Retain
+pvc-5ead43d1-3317-11ea-97b0-0aef8816c7ea ... Retain
+pvc-7e1f67f9-3317-11ea-a650-06e1eadd9a4c ... Retain
+pvc-7e21042e-3317-11ea-9786-02deaf9aa87e ... Retain
+pvc-7e226978-3317-11ea-97b0-0aef8816c7ea ... Retain
+
+CLAIM                                     STORAGECLASS
```

Comment:
Is there some way how we can put it all on the same line? It took me a while to realize these two columns were on the same line as the previous ones.

Reply:
I dropped the StorageClass column and put everything on a single line. I was trying to avoid the wrap/horizontal scroll.